.github/workflows/buildAndTest.yaml

```diff
@@ -28,5 +28,39 @@ jobs:
           --activate-profiles full-checkstyle \
           --show-version \
           --define maven.javadoc.skip=true \
-          --threads 1.5C \
           test
+
+  integrationTests:
+    if: github.repository == 'GoogleCloudPlatform/spring-cloud-gcp'
+    needs: unitTests
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Setup gcloud
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: latest
+          project_id: spring-cloud-gcp-ci
+          service_account_key: ${{ secrets.SPRING_CLOUD_GCP_CI_SA_KEY }}
+          export_default_credentials: true
+      - name: Integration Tests
+        run: |
+          ./mvnw \
+            --batch-mode \
+            --activate-profiles integration-tests \
+            --show-version \
+            --define test="**/*IntegrationTest*" \
```

Comment:
Just a heads up, I think there might be some integration tests that are not named correctly; might have to rename those tests as you expand to other modules.

Reply:
Ack - addressing in next PR.